org.scala-ide.sdt.aspects/META-INF/aop.xml

```diff
@@ -31,5 +31,6 @@
 <aspect name="scala.tools.eclipse.contribution.weaving.jdt.debug.JDIModelPresentationAspect"/>
 <aspect name="scala.tools.eclipse.contribution.weaving.pde.ui.PluginImportAspect"/>
 <aspect name="scala.tools.eclipse.contribution.weaving.jdt.launching.JUnitLaunchConfigurationTabAspect"/>
+<!-- aspect name="scala.tools.eclipse.contribution.weaving.jdt.jdi.JdiEntryGuardianAspect"/ -->
```

Comment:
I'm surprised this one isn't needed. Does it really apply the aspect when this line is commented? If so, just remove the line, no need to have stray comments here.